### DISCUSSION TITLE
Templates in custom outputs of a trigger: state node

### DISCUSSION THREAD

**Discussion by bonanitech** [zachowj/node-red-contrib-home-assistant-websocket#541] [Feature Request]
Is there a way to use templates in the custom outputs of a trigger: state node?

**Comment by zachowj** [comment]
No, the trigger-state node does need some updating in the output options. It's really lagging behind other nodes.

**Reply by bonanitech** [reply]
Thanks for the feedback, @zachowj. Should I mark this as an answer and create a feature request?

**Reply by zachowj** [reply]
I changed it to a feature request.
